packages/neuron-ui/src/containers/Main/hooks.ts

```diff
@@ -294,13 +296,17 @@ export const useSubscription = ({
         }
       }
     })
+    const showGlobalDialogSubject = ShowGlobalDialogSubject.subscribe(params => {
```

Comment:
I'l like to know why `show global dialog` and `show alert dialog` are paired

Reply:
Maybe renaming `showAlertDialog` to `showGlobalAlertDialog` is clear?

Reply:
It would be more clear and won't mislead other developers in the future